src/figures.jl

```diff
@@ -35,6 +35,7 @@ current_figure!(fig) = (_current_figure[] = fig)
 
 "Returns the current active axis (or the last axis that got created)"
 current_axis() = current_axis(current_figure())
+current_axis(::Nothing) = nothing
```

Comment:
This should probably just be a more descriptive error, something along the lines of "No axis exists yet, please create one first" or something.

Thanks for the PR!